modules/geo-layers/src/mesh-layer/mesh-layer-vertex.glsl.js

```diff
@@ -4,13 +4,15 @@ export default `#version 300 es
 // Scale the model
 uniform float sizeScale;
 uniform bool composeModelMatrix;
+uniform bool u_pickObjectIds;
 
 // Primitive attributes
 in vec3 positions;
 in vec3 normals;
 in vec3 colors;
 in vec2 texCoords;
 in vec4 uvRegions;
+in vec3 objectIdsPickingColors;
```

Comment:
Does it render without error if this attribute is never set? Have you tested a dataset without objectIds?

Reply:
If `pickFeatures` is enabled and there are no any `objectIds` only instance picking works because `u_pickObjectIds` flag is set to false.
I tested it by manually removing `objectIds` from tiles.